Misc/NEWS.d/next/Library/2021-04-08-22-11-27.bpo-25264.hEqA5d.rst

```diff
@@ -0,0 +1,6 @@
+:func:`os.path.realpath` now raises :exc:`OSError` when a symlink loop is
+encountered. Previously it returned a path with unresolved symlinks.
+
+:func:`os.path.realpath` now accepts a *strict* keyword-only argument.
```

Comment:
I think this needs to be a single paragraph. Alternatively, you can add a second NEWS file by copying the name and just changing the random letters in the last part of the filename.